### DISCUSSION TITLE
Estimate Time For Running Standard Notebooks Of Crop Segmentation Using Recommended VM Specifications

### DISCUSSION THREAD

**Discussion by saqibzia-dev** [microsoft/farmvibes-ai#34] [Q&A]
Hi,
 I am running standard crop segmentation notebooks (without making any changes) on virtual machine with recommended specifications. Can you please provide the estimate time each notebook will take to complete?

**Comment by rafaspadilha** [comment]
Hi, @saqibzia-dev. Thank you for using FarmVibes.AI and sorry for the late reply.
We are computing updated run time and required disk space for running the Crop Segmentation examples and will update here (and directly on the notebooks) in the next few days.
